db/migrate/20231130071816_create_posts.rb

```diff
@@ -1,7 +1,7 @@
 class CreatePosts < ActiveRecord::Migration[7.1]
   def change
     create_table :posts do |t|
-      t.references :author, null: false, foreign_key: true
+      t.references :author, null: false, foreign_key: { to_table: :users } 
       t.string :title
       t.text :text
       t.integer :comments_counter
```

Comment:
- You have done well so far :+1:  but there are a few changes you need to make before moving to the next milestone. :+1: :100: 

- - Error screenshot:
![Selection_050](https://github.com/demesameneshoa/blog-app/assets/107632780/2f026c33-e34d-438d-a64a-d4da8a0e7d6a)


- [ ] When I tried setting up your project to run the tests, I Got this error when I tried to migrate your database. The error indicates that the 'comments' migration is trying to reference a 'posts' table that hasn't been created yet. :+1: :+1: 

In Rails, the order of migration files matters, especially when there are associations between tables. Since a comment belongs to a post, the `posts` table should be created before the `comments` table.

To resolve this, you can either:

Change the timestamp in the `post` migration file `20231130071816_create_posts.rb` to a value earlier than the `comments and likes` migration file.
or
Rewrite the migration files so that the 'posts' table comes before the 'comments' table in the migration order.
:smile: :+1:  I'll suggest you rectify this because a user can't setup your project to run the test with these errors. :small_airplane: :smile: